### PR TITLE
Extend e2e tests to cover mil4 use case

### PR DESCRIPTION
**Pull request by ognjenkurtic**
# Description

Extends the existing e2e test to assert state storage after SRI use-case workstep1 exection. Please review after history tree PR and workstep PRs have been merged.

## Related Issue

#729 

## Motivation and Context

In order to validate the execution of a single workstep from end to end

## How Has This Been Tested

By running the e2e tests locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Request to be added as a Code Owner/Maintainer

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Comment by ognjenkurtic** [comment]
@biscuitdey @Kasshern @Therecanbeonlyone1969 @Ybittan Rebased main, ready for review

**Comment by ognjenkurtic** [comment]
Merged from main but lett's wait for Shrees PR to be merged before we review this

File: examples/bri-3/src/bri/communication/api/messages.controller.spec.ts [hunk 1]
```diff
@@ -34,6 +34,7 @@ import {
 } from './err.messages';
 import { MessageController } from './messages.controller';
 import { mockDeep, DeepMockProxy } from 'jest-mock-extended';
+import { ethers } from 'ethers';
```
Comment (Kasshern) [inline examples/bri-3/src/bri/communication/api/messages.controller.spec.ts:37]: remove unused import

File: examples/bri-3/src/bri/identity/bpiAccounts/api/accounts.controller.spec.ts [hunk 2]
```diff
@@ -2,36 +2,41 @@ import { BadRequestException, NotFoundException } from '@nestjs/common';
...
-import { uuid } from 'uuidv4';
+import { AccountController } from './accounts.controller';
+import { CreateBpiAccountDto } from './dtos/request/createBpiAccount.dto';
+import { NOT_FOUND_ERR_MESSAGE } from './err.messages';
 
 describe('AccountController', () => {
   let accountController: AccountController;
   let accountStorageAgentMock: DeepMockProxy<BpiAccountStorageAgent>;
   let subjectAccountStorageAgentMock: DeepMockProxy<BpiSubjectAccountStorageAgent>;
+  let merkleTreeStorageAgentMock: DeepMockProxy<MerkleTreeStorageAgent>;
```
Comment (Kasshern) [inline examples/bri-3/src/bri/identity/bpiAccounts/api/accounts.controller.spec.ts:39]: remove unused variable

File: examples/bri-3/src/bri/identity/bpiAccounts/capabilities/getBpiAccountById/getBpiAccountByIdQuery.handler.ts [hunk 3]
```diff
@@ -1,10 +1,12 @@
 import { Mapper } from '@automapper/core';
 import { InjectMapper } from '@automapper/nestjs';
 import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { MerkleTreeStorageAgent } from '../../../../merkleTree/agents/merkleTreeStorage.agent';
 import { BpiAccountStorageAgent } from '../../agents/bpiAccountsStorage.agent';
 import { BpiAccountDto } from '../../api/dtos/response/bpiAccount.dto';
 import { BpiAccount } from '../../models/bpiAccount';
 import { GetBpiAccountByIdQuery } from './getBpiAccountById.query';
+import { stat } from 'fs';
```
Comment (Kasshern) [inline examples/bri-3/src/bri/identity/bpiAccounts/capabilities/getBpiAccountById/getBpiAccountByIdQuery.handler.ts:9]: remove unused variable

**Review by Kasshern — APPROVED** [review]

**Review by biscuitdey — APPROVED** [review]
Approving.

**Pull request merged**